### DISCUSSION TITLE
Transcript generation seems to pause when people talk over each other

### DISCUSSION THREAD

**Discussion by lorikku** [deepgram/community#687] [General help]
I've tested transcript generation with Deepgram a few times now, and so far so good, until yesterday.

I was in a meeting where people were talking over each other a lot, with the resulting transcript having quite a few missing pieces, specifically the moments when people were talking over each other.

Is there something I can do about this? I can imagine not every meeting would have every person wait their turn to talk.

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.
